documentation/book/con-overview-of-kafka-bridge.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-kafka-bridge.adoc
+
+[id='con-overview-of-kafka-bridge-{context}']
+= Overview of the {ProductName} Kafka Bridge
+
+The {ProductName} Kafka Bridge provides a REST API for integrating HTTP-based clients with a Kafka cluster running on {ProductPlatformName}. You can use it to integrate clients that do not have a Kafka client library and so cannot transfer data using the native Kafka protocol.
+
+The Kafka Bridge API consists of two resources -- `consumers` and `topics` -- that together provide several endpoints for interacting with Kafka consumers and producers. You can use the API to: 
+
+* Send messages to a topic or partition.
```

Comment:
When you send message to a topic, you are always sending message to a partition in that topic so the "or" should not be there.

Reply:
That's why my brain seems to prefer the term "topic partition"! :D

I'll change to "topic" in all cases like this.